samples/simulated_device_cloud_upload/README.md

```diff
@@ -59,6 +59,36 @@ Windows
 
 >run `ctest -C Debug -V`
 
+### Yocto Linux and Intel Edison
+
+We recommend using the Azure recipes provided by Intel available in the 
+[meta-iot-cloud](https://github.com/intel-iot-devkit/meta-iot-cloud) repository.
+Follow the instructions in the 
+[README](https://github.com/intel-iot-devkit/meta-iot-cloud#configuration) to 
+build an Edison image with the azure-iot-gateway-sdk libraries included.
+
+#### Intel Edison
+
+The Edison does not support all of the Azure recipes available in the "meta-iot-cloud" repository. To get the Edison image to build, we have to limit the recipes included in the image.
+
+- Remove the azure-iot-gateway-sdk-java target.
+
+    The dependencies for Java require libraries not included in the Edison BSP 
+    source package. To avoid building the java target, add the following line 
+    to your `conf/local.conf` file.
+
+```
+PACKAGECONFIG_pn-azure-iot-gateway-sdk = "bluetooth"
```

Comment:
It's not obvious, sorry.  There are 3 packages in the azure-iot-gateway-sdk recipe: the main package, azure-iot-gateway-sdk-java and azure-iot-gateway-sdk-bluetooth.  This statement will limit the packages to the main and bluetooth packages.  I'll try to make this more clear.